packages/ocs/dashboards/object-service/details-card/details-card.tsx

```diff
@@ -136,6 +161,19 @@ export const ObjectServiceDetailsCard: React.FC<{}> = () => {
           >
             {serviceVersion}
           </DetailItem>
+          <DetailItem
+            key="encryption"
+            title={t('Encryption')}
+            isLoading={!ocsLoaded}
+            error={ocsError}
+          >
+            <EncryptionPopover
+              encryptionDetails={encryptionDetails}
+              hasStorageClassEncryption={hasStorageClassEncryption}
+              inTransitEncryption={false}
+              hasMCGEncryption={hasMCG}
```

Comment:
`hasMCG` tell u whether NooBaa CR is existing in the namespace or not...
and from name `hasMCGEncryption` it suggests this flag signifies whether encryption is enabled or not...

both flags are independent of each others... either assumption here is wrong or naming is wrong... 
```suggestion
              hasMCGEncryption={hasMCG}
```